test/reputation-system/root-hash-submissions.js

```diff
@@ -216,6 +221,10 @@ contract("Reputation mining - root hash submissions", (accounts) => {
       const miningSkillId = 3;
 
       await metaColony.setReputationMiningCycleReward(WAD.muln(10));
+
+      // Need tokens to pay out rewards
+      await giveUserCLNYTokens(colonyNetwork, colonyNetwork.address, WAD.muln(100));
```

Comment:
Why is this needed now?

Reply:
The chainId we're testing on is now not one where we can mint tokens on payouts.